model/stream.go

```diff
@@ -108,3 +114,17 @@ func (s Stream) FriendlyDate() string {
 func (s Stream) FriendlyTime() string {
 	return s.Start.Format("02.01.2006 15:04") + " - " + s.End.Format("15:04")
 }
+
+func (s Stream) FriendlyNextDate() string {
+	nowy, nowm, nowd := time.Now().Date()
+	tmrwy, tmrwm, tmrwd := time.Now().Add(time.Hour * 24).Date()
+	starty, startm, startd := s.Start.Date()
+
+	if nowy == starty && nowm == startm && nowd == startd {
+		return "Today,"
+	} else if tmrwy == starty && tmrwm == startm && tmrwd == startd {
+		return "Tomorrow,"
+	}
+
+	return fmt.Sprintf("%v %02d.", s.Start.Month(), s.Start.Day())
+}
```

Comment:
Using jinzhu/now could improve readability: 
```suggestion
func (s Stream) FriendlyNextDate() string {
	if now.With(s.Start).EndOfDay() == now.EndOfDay() {
		return "Today,"
	}
	if now.With(s.Start).EndOfDay() == now.With(time.Now().Add(time.Hour*24)).EndOfDay() {
		return "Tomorrow,"
	}
	return fmt.Sprintf(s.Start.Format("Mon, January 02."))
}
```
Also, this adds the day of the week to the date.

Reply:
That's great. Thanks!

Reply:
Oh and this function is exported, it should have some documentation (shame on me for not always doing this myself :D)